src/main/resources/ca/corefacility/bioinformatics/irida/database/changesets/19.04/analysis-templates.xml

```diff
@@ -13,13 +13,37 @@
                 <constraints foreignKeyName="FK_ANALYSIS_TEMPLATE_PROJECT" referencedColumnNames="id"
                              referencedTableName="project"/>
             </column>
+            <column name="automated" type="bit(1)" defaultValue="0"/>
```

Comment:
I believe in theory `automated` should default to `NULL` here since it doesn't apply for `AnalysisSubmissionTemplate`. I don't think it really makes a difference to the operation of the code though so I don't know if I'm being overly-critical.

Reply:
Yeah should never happen in code, but since they're sharing a table now I guess it's worth it.  Update in 6aca41afa1.  Had to update all the test files that included `analysis_submission`.

Reply:
Thanks. Looks great.